neqo-http3/src/transaction_server.rs

```diff
@@ -333,12 +333,10 @@ impl Http3Transaction for TransactionServer {
         matches!(self.send_state, TransactionSendState::SendingResponse { .. })
     }
 
-    fn is_state_sending_data(&self) -> bool {
-        self.has_data_to_send()
+    fn reset_receiving_side(&mut self) {
+        self.recv_state = TransactionRecvState::Closed;
```

Comment:
Did you add a test case for this?

Reply:
I added this to be completely correct, but anytime rest_receiving_side is called on the server side we also remove transaction, so we cannot actually test this, it is also fine not to set the state.